tensorflow/lite/kernels/activations.cc

```diff
@@ -1113,22 +1089,49 @@ TfLiteStatus LeakyReluEval(TfLiteContext* context, TfLiteNode* node) {
   const LeakyReluOpData* data =
       reinterpret_cast<LeakyReluOpData*>(node->user_data);
 
-  LeakyReluParams op_params;
-  op_params.alpha = params->alpha;
   switch (input->type) {
     case kTfLiteFloat32: {
+      LeakyReluParams op_params{params->alpha};
       optimized_ops::LeakyRelu(
           op_params, GetTensorShape(input), GetTensorData<float>(input),
           GetTensorShape(output), GetTensorData<float>(output));
       return kTfLiteOk;
     } break;
-    case kTfLiteUInt8: {
-      QLeakyRelu<uint8_t>(input, output, params->alpha, data);
+    case kTfLiteUInt8:
+    {
```

Comment:
can you use clang_format to format the file?

Reply:
clang-formated all the files touched in the PR.